src/Facade.php

```diff
@@ -106,6 +106,41 @@ public function getTicketHtml($hash)
         return $gateway->getTicketHtml($hash);
     }
 
+    /**
+     * @param string $key
+     *
+     * @return bool
+     */
+    public function isValidPrivateKey($key)
+    {
+        $data = ['integration_key' => $key];
+        $response = $this->getHttpClient()->validatePrivateKey($data);
+
+        return $response['status'] === 'SUCCESS';
+    }
+
+    /**
+     * @param string $key
+     *
+     * @return bool
+     * @throws \Exception
+     */
+    public function isValidPublicKey($key)
+    {
+        $data = ['public_integration_key' => $key];
+        try {
+            $response = $this->getHttpClient()->validatePublicKey($data);
+
+            return $response['status'] === 'SUCCESS';
+        } catch (\Exception $e) {
```

Comment:
What kind of exception could happen in server-side besides the key being invalid and getting a 200 with error status?